.github/workflows/phpunit.yaml

```diff
@@ -0,0 +1,29 @@
+# .github/workflows/phpunit.yaml
+name: phpunit
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['5.6', '7.0', '7.1']
```

Comment:
Don't we need more than 7.1, like: 7.2, 7.3, 7.3 and 8.0? Do we still need 5.6?

Reply:
I've removed 5.6, and added 7.2, 7.3 and 7.4. It won't work with PHP 8.0 yet because of some of the composer dependencies. 

A PHP8 upgrade would need to be it's own separate ticket, it would also require a bit of careful thought because we have to consider how it would work with phpbb/customisation-db (as that project depends on this one), similar to how we ran into issues when we upgraded this project from Symfony 2.8